From 9ce58e46f935dae9db1086ca029d97843c533a3c Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Thu, 18 Dec 2025 10:01:07 +0100
Subject: [PATCH] python-distro: add dependency on setuptools

Package python-distro compiled with error with missing setuptools
Fixing:
pyproject_hooks._impl.BackendUnavailable: Cannot import 'setuptools.build_meta'
---
 lang/python/python-distro/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lang/python/python-distro/Makefile b/lang/python/python-distro/Makefile
index 9312ba7d5..6e87c9e1b 100644
--- a/lang/python/python-distro/Makefile
+++ b/lang/python/python-distro/Makefile
@@ -11,6 +11,8 @@ PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
 
+PKG_BUILD_DEPENDS:=python-setuptools/host
+
 include ../pypi.mk
 include $(INCLUDE_DIR)/package.mk
 include ../python3-package.mk
-- 
2.52.0

